### DISCUSSION TITLE
Attachment not displayed in approved bookings

### DISCUSSION THREAD

**Discussion by kevin-vanb** [liedekef/events-made-easy#677] [General]
On our event submission we require an attachment with a drawing of table setup. In the past this worked just fine, we could see the submitted images and download them in the approved bookings, but in current version we don't see them anymore.
Any idea on this?
EME Version 2.5.18
WP 6.6.2

**Comment by liedekef** [comment]
Works fine here. Make sure the corresponding column is made visible in the bookings overview.

**Reply by kevin-vanb** [reply]
Hello Franky, took some time to figure it out, but I think I found the issue.
The custom fields were already added, but remained empty even after uploading a random file again in the backend of eme (approved bookings form)
Is there a restriction on the filetype that can be uploaded? I tested with jpg and pdf, which works fine, it fails with png, jpeg and docx for example.

Kind regards,
Kevin

**Comment by liedekef** [comment]
ok, I figured it out: I'm guessing your filenames contain "-" in them (dashes, not underscores) and now I allow those but it messed up an explode-call :-)
Go in eme-functions.php and change line 3100 from
`                    $info = explode( '-', $entry );
`
to
`                    $info = explode( '-', $entry, 4 );
`
That should fix it (and the uploads were ok btw, otherwise you would've received an upload error)

**Reply by kevin-vanb** [reply]
Great work Franky, thanks for the workaround and the fix (I just installed it : -) )
